pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -850,11 +850,10 @@ func ApplyRingHashLoadBalancer(c *cluster.Cluster, lb *networking.LoadBalancerSe
 func applyLocalityLBSetting(locality *core.Locality, proxyLabels map[string]string, cluster *cluster.Cluster,
 	localityLB *networking.LocalityLoadBalancerSetting,
 ) {
-	// Failover should only be applied with outlier detection, or traffic will never failover.
-	enabledFailover := cluster.OutlierDetection != nil
+	// Failover should only be applied with outlier detection or SendUnhealthyEndpoints, or traffic will never failover.
+	odEnabled := cluster.OutlierDetection != nil
 	if cluster.LoadAssignment != nil {
-		// TODO: enable failoverPriority for `STRICT_DNS` cluster type
```

Comment:
Why is this TODO removed?

Reply:
My fault probably, this comment was a bit misleading to me, as the following code that applies locality failover settings is always called for both STRICT_DNS and EDS clusters, so it's not immediately clear what exactly is TODO.

On second thought, STRICT_DNS clusters are only generated from ServiceEntries and in that case we need API support in the CRD to explicitly set localities for specified endpoints (or we would have no way to know about them), but I'm not sure if there are any plans to implement that. So probably should comment about that instead, for clarity.